c++/src/CMakeLists.txt

```diff
@@ -125,6 +125,83 @@ include_directories (
   ${LZ4_INCLUDE_DIRS}
   )
 
+# To avoid reading the Timezone database from disk, we load the file during the
+# build and inject them as binary into the library. This can increase the size
+# of library but avoids doing system calls at runtime. The behavior can be
+# disabled by using the CMake flag NO_EMBEDDED_TZ_DB
+if (NOT DEFINED EMBEDDED_TZ_DB)
+  set(EMBEDDED_TZ_DB ON)
```

Comment:
A side effect of this is when the time zone info files are updated, the Orc lib has to be recompiled and redistributed.